de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/modelvalidator/M_ShipmentSchedule_QtyPicked.java

```diff
@@ -62,4 +73,37 @@ public void invalidateShipmentSchedule(final I_M_ShipmentSchedule_QtyPicked ship
 
 		Services.get(IShipmentSchedulePA.class).invalidate(singletonList, trxName);
 	}
+
+	@ModelChange(timings = {
+			ModelValidator.TYPE_BEFORE_NEW,
+			ModelValidator.TYPE_BEFORE_CHANGE })
+	public void validateOverdelivery(final I_M_ShipmentSchedule_QtyPicked schedQtyPicked)
```

Comment:
pls take it out. QtyToDeliver is updated on server side so at this moment, might be stale.